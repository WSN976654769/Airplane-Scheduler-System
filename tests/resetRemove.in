location add Mumbai 19.08 72.741 0.4
location add NewDelhi 28.527 77.0688988 -0.123
flight add monday 08:00 mumbai newdelhi 120
flight 0
flight 0 book 23
flight 0 reset
flight 0
flight 0 book 130
flight 0 remove
flights
exit
