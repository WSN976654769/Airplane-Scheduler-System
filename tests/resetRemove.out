User: Successfully added location Mumbai.

User: Successfully added location NewDelhi.

User: Successfully added Flight 0.

User: Flight 0
Departure:    Mon 08:00 Mumbai
Arrival:      Mon 09:35 NewDelhi
Distance:     1,139km
Duration:     1h 35m
Ticket Cost:  $317.79
Passengers:   0/120

User: Booked 23 passengers on flight 0 for a total cost of $7041.08

User: Reset passengers booked to 0 for Flight 0, Mon 08:00 Mumbai --> NewDelhi.

User: Flight 0
Departure:    Mon 08:00 Mumbai
Arrival:      Mon 09:35 NewDelhi
Distance:     1,139km
Duration:     1h 35m
Ticket Cost:  $317.79
Passengers:   0/120

User: Booked 120 passengers on flight 0 for a total cost of $36255.03
Flight is now full.

User: Removed Flight 0, Mon 08:00 Mumbai --> NewDelhi, from the flight schedule.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
(None)

User: Application closed.
